src/seam-connect/client.ts

```diff
@@ -1,13 +1,24 @@
-import axios, { AxiosInstance, AxiosRequestConfig } from "axios"
+import axios, {
+  AxiosInstance,
+  AxiosRequestConfig,
+  AxiosRequestHeaders,
+} from "axios"
 import axiosRetry from "axios-retry"
 import { SeamAPIError, SeamMalformedInputError } from "../lib/api-error"
 import { Routes } from "./routes"
-import { ErroredAPIResponse, SuccessfulAPIResponse } from "../types/globals"
+import {
+  APIResponse,
+  ErroredAPIResponse,
+  SuccessfulAPIResponse,
+} from "../types/globals"
 import { version } from "../../package.json"
+import { ClientSessionResponseInterface } from "../types"
 
 export interface SeamClientOptions {
   /* Seam API Key */
   apiKey?: string
+  /* Seam Client Access Token */
+  clientAccessToken?: string
```

Comment:
I noticed in the new seam provider, we just pass all the keys as `apiKey` even if it's a `clientAccessToken`.

Do we still want a new option? Or just have `apiKey`, let the user pass whatever token they have, and then use the prefix if we need to do special logic?

Reply:
i think we should be generous in acceptance (since it's detectable) but we shouldn't conflate the options